demisto_sdk/commands/common/update_id_set.py

```diff
@@ -799,6 +799,48 @@ def get_widget_data(path):
     return {id_: data}
 
 
+def get_dashboard_data(path):
+    data = OrderedDict()
+    json_data = get_json(path)
+    layouts = json_data.get('layout', {})
+    return parse_dashboard_report_data(layouts, data, json_data, path)
+
+
+def get_report_data(path):
+    data = OrderedDict()
+    json_data = get_json(path)
+    layouts = json_data.get('dashboard', {}).get('layout')
+    return parse_dashboard_report_data(layouts, data, json_data, path)
+
+
+def parse_dashboard_report_data(layouts, data, json_data, path):
```

Comment:
Also, why aren't we giving the variables better name? because it's like that throughout the code?
```suggestion
def parse_dashboard_or_report_data(layouts, data, json_data, path):
```

Reply:
Done